app/Http/Controllers/Api/V1/ActivityController.php

```diff
@@ -166,6 +166,7 @@ public function store(ActivityCreateRequest $request, Playlist $playlist)
         $data = $request->validated();
 
         $data['order'] = $this->activityRepository->getOrder($playlist->id) + 1;
+        $data['shared'] = Project::where('id', $playlist->project_id)->value('shared');
```

Comment:
The "shared" for the project can be get using following code

$playlist->project->shared